packages/xo-server/src/xo-mixins/rest-api.mjs

```diff
@@ -225,14 +226,25 @@ async function _getDashboardStats(app) {
   }
 
   try {
-    const backupRepositoriesSize = Object.values(await app.getAllRemotesInfo()).reduce(
-      (prev, remoteInfo) => ({
-        available: prev.available + remoteInfo.available,
-        backups: 0, // @TODO: compute the space used by backups
-        other: 0, // @TODO: compute the space used by everything that is not a backup
-        total: prev.total + remoteInfo.size,
-        used: prev.used + remoteInfo.used,
-      }),
+    const remotes = await app.getAllRemotes()
+    const remotesInfo = await app.getAllRemotesInfo()
+
+    const backupRepositoriesSize = remotes.reduce(
+      (prev, remote) => {
+        const { type } = parse(remote.url)
+        if (!remote.enabled || type === 's3') {
+          return prev
+        }
```

Comment:
s3 backups will have backups size ( and maybe other sizes ) , maybe it could be useful to split this test : if remote is disable skip, if remote is s3 then available is 0

Reply:
`backupRepositoriesSize` is used by the frontend to display all non `S3 backup repositories` size.

I think we will have to revisit how to build `backupRepositoriesSize` later when there will be a need to display values ​​for S3, like something `backupRepositoriesSize = { s3: ..., other: ...}`.

But I suggest not to implement a breaking change just before the release